From FPGA

tienfeek opened the pull request:
FPGA support quantitative model

MyPandaShaoxiang committed:
fix: move some necessary op from extra to basic
test=develop

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/Paddle-Lite?pullRequest=2883) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/Paddle-Lite?pullRequest=2883) before we can accept your contribution.<br/>**2** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: MyPandaShaoxiang<br/>:white_check_mark: chonwhite<br/>:x: tienfeek<br/>:x: root<br/><hr/>**root** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/Paddle-Lite?pullRequest=2883) it.</sub>

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/Paddle-Lite?pullRequest=2883) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/Paddle-Lite?pullRequest=2883) before we can accept your contribution.<br/>**2** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: chonwhite<br/>:white_check_mark: MyPandaShaoxiang<br/>:x: root<br/>:x: tienfeek<br/><hr/>**root** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/Paddle-Lite?pullRequest=2883) it.</sub>

tienfeek committed:
FPGA support quantitative model
test=develop

MyPandaShaoxiang commented on lite/api/CMakeLists.txt:
```diff
@@ -223,14 +223,24 @@ if(LITE_WITH_LIGHT_WEIGHT_FRAMEWORK AND WITH_TESTING)
+       DEPS ${lite_model_test_DEPS}
+       CL_DEPS ${opencl_kernels}
+       FPGA_DEPS ${fpga_kernels})
+
```
不存在该文件吧，建议去掉该项，

MyPandaShaoxiang commented on lite/api/CMakeLists.txt:
```diff
@@ -223,14 +223,24 @@ if(LITE_WITH_LIGHT_WEIGHT_FRAMEWORK AND WITH_TESTING)
-   # lite_cc_test(test_ocr_attention SRCS ocr_attention_test.cc
-   #    DEPS ${lite_model_test_DEPS})
+   lite_cc_test(test_ocr_attention_fpga SRCS ocr_attention_test_fpga.cc
+      DEPS ${lite_model_test_DEPS})
```
同上

MyPandaShaoxiang commented on lite/core/program.cc:
```diff
@@ -139,6 +139,9 @@ void RuntimeProgram::Run() {
     if (inst.is_feed_fetch_op()) continue;
+    std::string op_type = inst.op()->op_info()->Type();
+    VLOG(4) << ">> Running kernel: " << inst.op()->op_info()->Repr()
+            << " on Target " << TargetToStr(inst.kernel()->target());
```
VLOG去掉吧，可能影响性能

MyPandaShaoxiang commented on lite/gen_code/paddle_infer.h:
```diff
@@ -46,7 +46,7 @@ class Tensor {
 class PaddlePredictor {
  public:
-  void Init();
+  void Init() {}
```
去掉这个吧

MyPandaShaoxiang commented on lite/kernels/host/CMakeLists.txt:
```diff
@@ -4,6 +4,7 @@ add_kernel(feed_compute_host Host basic SRCS feed_compute.cc DEPS ${lite_kernel_
 add_kernel(fetch_compute_host Host basic SRCS fetch_compute.cc DEPS ${lite_kernel_deps})
 add_kernel(reshape_compute_host Host basic SRCS reshape_compute.cc DEPS ${lite_kernel_deps} reshape_op)
 add_kernel(multiclass_nms_compute_host Host basic SRCS multiclass_nms_compute.cc DEPS ${lite_kernel_deps})
+add_kernel(one_hot_compute_host Host basic SRCS one_hot_compute.cc DEPS ${lite_kernel_deps})
```
这个改成extra吧

tienfeek committed:
delete unused code test=develop

tienfeek committed:
delete unused code test=develop
delete debugger info
 Please enter the commit message for your changes. Lines starting

Commit:
Merge branch 'new_dev' of https://github.com/PaddleLite-EB/Paddle-Lite into fpga_pr

tienfeek committed:
Merge branch 'new_dev' of https://github.com/PaddleLite-EB/Paddle-Lite into new_dev

tienfeek committed:
Merge branch 'new_dev' of https://github.com/PaddleLite-EB/Paddle-Lite into new_dev
delete debugger include

tienfeek committed:
Merge branch 'new_dev' of https://github.com/PaddleLite-EB/Paddle-Lite into new_dev

MyPandaShaoxiang committed:
fix: fix compile problem

MyPandaShaoxiang committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/Paddle-Lite into fpga_pr

tienfeek committed:
Merge branch 'fpga_pr' into new_dev

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/Paddle-Lite?pullRequest=2883) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/Paddle-Lite?pullRequest=2883) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: MyPandaShaoxiang<br/>:white_check_mark: tienfeek<br/>:x: root<br/><hr/>**root** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/Paddle-Lite?pullRequest=2883) it.</sub>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/Paddle-Lite?pullRequest=2883) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/Paddle-Lite?pullRequest=2883) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: tienfeek<br/>:white_check_mark: MyPandaShaoxiang<br/>:x: root<br/><hr/>**root** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/Paddle-Lite?pullRequest=2883) it.</sub>
